instrumentation/aws-lambda/aws-lambda-core-1.0/javaagent/README.md

```diff
@@ -0,0 +1,5 @@
+# Settings for the AWS Lambda Instrumentation
+
+| System property                                 | Type    | Default | Description                    |
+|-------------------------------------------------|---------|---------|--------------------------------|
+| `otel.instrumentation.aws-lambda.flush-timeout` | Integer | 10000   | Flush timeout in milliseconds. |
```

Comment:
How about using `Long` instead of `Integer` here?

Reply:
Does it really matter? From user perspective I'd assume it is only important to know that a numeric value is expected.